eth/executionclient/defaults.go

```diff
@@ -8,8 +8,12 @@ const (
 	DefaultConnectionTimeout           = 10 * time.Second
 	DefaultReconnectionInitialInterval = 1 * time.Second
 	DefaultReconnectionMaxInterval     = 64 * time.Second
+	DefaultHealthInvalidationInterval  = 10 * time.Second // TODO: decide on this value, for now choosing a bit less than block interval
```

Comment:
this should be double the interval of NodeProber (i think 1 slot) to let it finish its own requests and avoid doing two in parallel

Reply:
`NodeProber`'s interval is 24 seconds, so I thought it needed to be done more often. There should not be parallel requests because the healthiness is cached for a short time